code/game/area/areas.dm

```diff
@@ -60,8 +60,8 @@
 	var/list/firealarms
 	var/firedoors_last_closed_on = 0
 
-	///Boolean to limit the areas (subtypes included) that atoms in this area can smooth with. Used for shuttles.
-	var/area_limited_icon_smoothing = FALSE
+	///Type path to limit the areas (subtypes included) that atoms in this area can smooth with. Used for shuttles.
+	var/area/area_limited_icon_smoothing = FALSE
```

Comment:
```suggestion
	var/area/area_limited_icon_smoothing = null
```

Reply:
Ya that was a stupid mistake, fixing it.